framework/encode/vulkan_state_writer.cpp

```diff
@@ -1368,7 +1369,7 @@ void VulkanStateWriter::ProcessImageMemory(const DeviceWrapper*
                 }
             }
         }
-        else
+        else if (image_wrapper->is_swapchain == false)
```

Comment:
`if (!image_wrapper->is_swapchain)`

Reply:
Fixed. Also changed `is_swapchain` to `is_swapchain_image`